src/main/kotlin/lotto/domain/LottoStore.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.domain
+
+class LottoStore {
+
+    fun purchaseAuto(money: Int): Lottoes {
+        val quantity = money / LOTTO_COST
+        val lottoes = arrayListOf<Lotto>()
+        repeat(quantity) {
+            lottoes.add(Lotto.generateAuto())
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val lottoes = (1..quantity).map { Lotto.generateAuto() }
```